<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Control AGV - TFG Marc Nueno</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Manual AGV</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnUp">
        <property name="text">
         <string>↑</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnDownRight">
        <property name="text">
         <string>↘</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnLeft">
        <property name="text">
         <string>←</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnDownLeft">
        <property name="text">
         <string>↙</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnDown">
        <property name="text">
         <string>↓</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnUpRight">
        <property name="text">
         <string>↗</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnRight">
        <property name="text">
         <string>→</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnUpLeft">
        <property name="text">
         <string>↖</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>241</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Vel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="velSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="singleStep">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Info Conexión</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ROS_MASTER_IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NODE IP *:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado: Conectado</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>541</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Consola</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Menú</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="monitorTab">
      <attribute name="title">
       <string>Monitorización</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="monitorLaserBtn">
          <property name="text">
           <string>Monitor Laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="monitorLidarBtn">
          <property name="text">
           <string>Monitor Lidar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="monitorMapBtn">
          <property name="text">
           <string>Posición Estimada y Objetivos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="monitorVideoBtn">
          <property name="text">
           <string>Feed Vídeo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="monitorOdomBtn">
          <property name="text">
           <string>Odometría &amp; Velocidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="actionTab">
      <attribute name="title">
       <string>Acciones</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="actionSlamBtn">
          <property name="text">
           <string>Generar Mapa SLAM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="actionOccMapBtn">
          <property name="text">
           <string>Mapa de Ocupación Binaria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="actionPlannerBtn">
          <property name="text">
           <string>Actualizar Planificador</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="actionObjectivesBtn">
          <property name="text">
           <string>Ver / Modificar Pedidos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="actionStockBtn">
          <property name="text">
           <string>Ver / Modificar Stock</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConexi_n">
    <property name="title">
     <string>Conexión</string>
    </property>
    <addaction name="actionRobot"/>
    <addaction name="actionROSbag"/>
   </widget>
   <addaction name="menuConexi_n"/>
  </widget>
  <action name="slamAction">
   <property name="text">
    <string>SLAM</string>
   </property>
  </action>
  <action name="actionRobot">
   <property name="text">
    <string>Robot</string>
   </property>
  </action>
  <action name="actionROSbag">
   <property name="text">
    <string>ROSbag</string>
   </property>
  </action>
  <action name="lidarAction">
   <property name="text">
    <string>LIDAR</string>
   </property>
  </action>
  <action name="actionAjustes">
   <property name="text">
    <string>Ajustes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
